config/services.yaml

```diff
@@ -51,14 +51,24 @@ services:
             - ["handle_paypal_payment_notification", "handle_paypal_membership_fee_payments"]
             - "@logger"
 
-        # Inject the log path and route into the PayPal IPN Logger
+    # Inject the log path and route into the PayPal IPN Logger
```

Comment:
This comment is lying btw. (copypaste error) No need to change it in this PR, but we should change that at some point.

Reply:
Probably don't event need the comment